test/factories/serial.rb

```diff
@@ -0,0 +1,17 @@
+FactoryGirl.define do
+  factory :serial do
+    sequence(:title) {|n| "Serial #{n}" }
+    sequence(:slug) {|n| "serial#{n}" }
+    description { "#{title} description\n" }
+
+    trait :whatever do
+      site
+    end
+
+    trait :with_posts do
+      after :create do |serial, _evaluator|
+        serial.posts << build(:post, :whatever, site: serial.site)
```

Comment:
Ouch. [`Post#category`](https://github.com/bm-sms/daimon-news/blob/master/test/factories/post.rb#L10) is needed.